app/src/sharedTest/java/org/oppia/app/player/state/StateFragmentTest.kt

```diff
@@ -159,6 +184,61 @@ class StateFragmentTest {
     }
   }
 
+  @Test
+  fun testNumberInputView_displaySoftInputAccordingToInputType_fieldsAreVisible() {
+    ActivityScenario.launch(HomeActivity::class.java).use {
+      Espresso.onView(withId(R.id.play_exploration_button)).perform(click())
+      val placeholder = "Write digit here."
+      Espresso.onView(ViewMatchers.withHint(placeholder)).check(matches(isDisplayed()))
```

Comment:
You can import `Espresso.onView` globally and use `onView` directly.

Reference: https://github.com/oppia/oppia-android/pull/224/files

Reply:
okay